src/frontend/context/InngangsvilkårContext.ts

```diff
@@ -14,6 +14,11 @@ import {
     RessursSuksess,
 } from '../typer/ressurs';
 
+type RadState = 'åpen';
+interface RaderState {
+    [key: string]: RadState;
+}
+
```

Comment:
Burde denne ha state "lukket" også? Og at hver rad alltid har en state som finnes?

Reply:
Trengs det? Nå fjernes verdien hvis den lukkes

Reply:
Trengs ikke, det var et alternativ i stedet for å fjerne verdien om den lukkes. Da finnes raden alltid, men har heller status "Lukket" om den er lukket (samme som å ikke eksistere). Vet ikke om det er noen fordeler med å gjøre det på den måten da